apx/common/src/apx_nodeInfo.c

```diff
@@ -28,6 +28,7 @@ static void apx_nodeInfo_connectRequirePortInternal(apx_nodeInfo_t *self, int32_
 static void apx_nodeInfo_connectProvidePortInternal(apx_nodeInfo_t *self, int32_t portIndex, apx_node_t *requesterNode, apx_port_t *requirePort);
 static void apx_nodeInfo_disconnectRequirePortInternal(apx_nodeInfo_t *requesterNodeInfo, int32_t requesterPortIndex);
 static void apx_nodeInfo_disconnectProvidePortInternal(apx_nodeInfo_t *providerNodeInfo, int32_t providerPortIndex, apx_portref_t *portref);
+static inline bool apx_nodeInfo_isPortEntryOutside(const apx_portDataMapEntry_t* portEntry, uint32_t portDataLen);

```

Comment:
According to MSDN documentation the inline keyword is only available in c++ compiler. I would remove "inline" for now until we have tested all versions of visual studio C compilers (2013, 2015, 2017)

Reply:
visual studio seems to be c90 that does not ahve inline support according to:
https://en.wikipedia.org/wiki/Inline_function#Standard_support

should we use a "#ifdef _MSC_VER" and __inline extension that seems to be supported by all modern visual studio versions? Or should we just skip inline alltogether...

Reply:
removed the inline for now to support all compilers